prediction_market_agent/agents/logprobs_agent/deploy.py

```
@@ -0,0 +1,261 @@
+        if raw_logprobs is None:
+            return None
+            
+        confidence = self._extract_token_probabilities(
+            raw_logprobs=raw_logprobs, 
+            model_type=PredictionResponse, 
+            field_key="p_yes", 
+        )
+        p_yes: str = max(confidence.items(), key=lambda item: item[1])[0]
+        response_json = clean_json_response(prediction.data)
+
+        if p_yes == None:
```

Ruff (0.11.9)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)